services/paymaster/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fogo-paymaster"
-version = "0.5.0"
+version = "0.5.1"
```

Comment:
note that this was bumped while i was testing `sampo`. you may want to adjust this to a more appropriate version